lib/cosmos/gui/utilities/script_module_gui.rb

```diff
@@ -61,13 +61,13 @@ def ask_string(question, blank_or_default = false, password = false)
       return answer.to_s
     end
 
-    def prompt_dialog_box(title, message, icon = Qt::MessageBox::NoIcon)
+    def prompt_dialog_box(title, message)
       result = nil
       Qt.execute_in_main_thread(true, 0.05) do
         window = nil
         window = get_cmd_tlm_gui_window() if get_cmd_tlm_gui_window()
         msg = Qt::MessageBox.new(window)
-        msg.setIcon(icon)
+        msg.setIcon(Qt::MessageBox::Warning)
```

Comment:
Accessing the Qt Constant outside of GUI context was invoking method missing and potentially a problem (It raised my new exception).